pkg/controllers/nodeclass/validation.go

```diff
@@ -40,6 +60,137 @@ func (n Validation) Reconcile(ctx context.Context, nodeClass *v1.EC2NodeClass) (
 			fmt.Sprintf("%q tag does not pass tag validation requirements", offendingTag))
 		return reconcile.Result{}, reconcile.TerminalError(fmt.Errorf("%q tag does not pass tag validation requirements", offendingTag))
 	}
+	// Auth Validation
+	if nodeClass.StatusConditions().Get(v1.ConditionTypeSecurityGroupsReady).IsFalse() || nodeClass.StatusConditions().Get(v1.ConditionTypeAMIsReady).IsFalse() || nodeClass.StatusConditions().Get(v1.ConditionTypeInstanceProfileReady).IsFalse() || nodeClass.StatusConditions().Get(v1.ConditionTypeSubnetsReady).IsFalse() {
+		nodeClass.StatusConditions().SetFalse(v1.ConditionTypeValidationSucceeded, "DependenciesNotReady", "Waiting for SecurityGroups, AMIs, and InstanceProfiles")
+		// nolint:nilerr
```

Comment:
nit: I'm surprised that you need a nolint for this still -- you're not checking any errors above

Reply:
Looks like you resolved this but you didn't comment on the commit and didn't change the code. If you don't think there should be a change, then I'd add a comment here -- making the change is also fine and then you can just auto-resolve this commit